components/dialog/ConfirmationDialogContent.tsx

```diff
@@ -31,11 +32,13 @@ export default function ConfirmationDialogContent({
           <DialogPrimitive.Description className="text-neutral-500">{children}</DialogPrimitive.Description>
         </div>
       </div>
-      <div className="mt-5 sm:mt-4 sm:flex sm:flex-row-reverse">
-        <DialogClose onClick={onConfirm} className="btn btn-primary">
+      <div className="mt-5 sm:mt-4 sm:flex sm:flex-row-reverse gap-x-2">
+        <DialogClose as={Button} color="primary" onClick={onConfirm}>
           {confirmBtnText}
```

Comment:
if you want them optional
```suggestion
          {confirmBtnText ?? 'Confirm'}
```